bin/bump-version.rb

```diff
@@ -7,12 +7,12 @@
 end
 component = ARGV[0].to_sym
 
-unless `which gh` && $?.success?
+unless `which gh` && $CHILD_STATUS.success?
```

Comment:
rubocop

Reply:
I think we can leave this change out of the PR as it's unrelated, and the scripts are currently not checked by rubocop on CI